### PR TITLE
Add client errors guide - starting with store_list

### PR DESCRIPTION
**Pull request by matr1x-hackmud**
### Problem

The client has a variety of cryptic/strange errors that we as vets are familiar with, but are not easy to understand. It's important to capture that information on the wiki, as many are here to stay forever. 

### Context

This starts with store_list-related bugs, which seem to be the most common (and most gnomic due to steam) errors the client gives

### CODE REVIEW

File: docs/guides/misc/client_errors.mdx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+### STORE ERROR: account is too new
+
+New accounts may only make purchases via ((%Cstore_list%)) after 3 days.
+
+### STORE ERROR: unknown
+
+This error is typically indicative of an untrusted Steam account. Unfortunately, the reason for this error is often difficult to dianogse.
+
+- The Steam account usually must exist for at least 30 days
+- The Steam account needs to have purchased at least $5 of things on Steam, from a real credit/debit card
```
Comment (Fayti1703) [inline docs/guides/misc/client_errors.mdx:18]: This doesn't have to be a credit/debit card, using a different payment method seems to work fine as well (in my case PayPal, I _believe_ I was also able to get by using just Gift Cards? But I'm not sure about that part of it.)

Comment (matr1x-hackmud) [reply docs/guides/misc/client_errors.mdx:18]: I'll change to "a valid payment method"

File: docs/guides/misc/client_errors.mdx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+## store_list
+
+### STORE ERROR: account is too new
+
+New accounts may only make purchases via ((%Cstore_list%)) after 3 days.
+
+### STORE ERROR: unknown
+
+This error is typically indicative of an untrusted Steam account. Unfortunately, the reason for this error is often difficult to dianogse.
```
Comment (seanmakesgames) [inline docs/guides/misc/client_errors.mdx:15]:
```suggestion
This error is typically indicative of an untrusted Steam account. Unfortunately, the reason for this error is often difficult to diagnose.
```

File: docs/guides/misc/client_errors.mdx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+- The Steam account usually must exist for at least 30 days
+- The Steam account needs to have purchased at least $5 of things on Steam, from a valid payment method
+- Accounts occasionally seem to be 'blacklisted' by Steam. If this happens, the account cannot ever be trusted
+  - It is not clear if VAC/game bans count for this
+  - Steam support has historically not assisted in this matter. Creation of a new account is the only known solution thus far
```
Comment (seanmakesgames) [inline docs/guides/misc/client_errors.mdx:21]: Once a new account is created, please contact hackmud support by joining the discord and creating a *support* ticket. We will assist you in re-associating your hackmud account with the new steam account.

(or something)

We should document this on the wiki too-- porting the 'this is how you report issues' content.

**Review by seanmakesgames — COMMENTED** [review]
some stuff to add

File: docs/guides/misc/client_errors.mdx (hunk 3 above)
Comment (seanmakesgames) [reply docs/guides/misc/client_errors.mdx:21]: > We should document this on the wiki too-- porting the 'this is how you report issues' content.

but this is a different work item to tackle

**Review by seanmakesgames — APPROVED** [review]

**Comment by matr1x-hackmud** [comment]
Changes look good to me

**Pull request merged**